nativelib/src/main/scala/scala/scalanative/runtime/CZone.scala

```diff
@@ -0,0 +1,15 @@
+package scala.scalanative.runtime
```

Comment:
`CZone` is also used in `Array` in `scala.scalanative.runtime` so it can't be `private[safe]`. Still, I could move it to `scala.scalanative.safe`, what do you think?

Reply:
Now SafeZone is moved to `scala.scalanative` and the `safe` package doesn't exist anymore.

Reply:
Let me reopen this conversation. 
We still should be able to hide this internal API from the users, the common package for both  `scala.scalanative.memory.SafeZone` and `scala.scalanative.runtime.Array` would be `private[scalanative]`. This should be enough.